torch/csrc/jit/runtime/vararg_functions.cpp

```diff
@@ -39,6 +39,130 @@ void format(Stack& stack, size_t num_inputs) {
   push(stack, ss.str());
 }
 
+// IValue tags are intentionally private, so we need additional logic to cast
+// the IValue type to the specified format.
+void addFormattedArg(
+    char key,
+    const IValue& ival,
+    std::stringstream& ss,
+    int precision = 6) {
+  // TODO: Implement precison-based formatting
+  switch (key) {
+    case 'd':
+    case 'i': {
```

Comment:
I think these curly brackets are not necessary, they are only needed when you create new variables inside `case`.